### PR TITLE
DOC: Fix heading capitalization in doc/source/whatsnew - part4 (#32550)

### PR DESCRIPTION
**Pull request by cleconte987**
Add exceptions to the list in 'scripts/validate_rst_title_capitalization.py'

- [ ] Modify files in doc/source/whatsnew: v0.14.0.rst, v0.14.1.rst, v0.15.0.rst, v0.15.1.rst, v0.15.2.rst, v0.16.0.rst, v0.16.1.rst, v0.16.2.rst, v0.17.0.rst, v0.17.1.rst
- [ ] Modify files in scripts: validate_rst_title_capitalization.py

### CODE REVIEW

**Comment by gfyoung** [comment]
Looks reasonable to me.

**Review by gfyoung — APPROVED** [review]

**Comment by jreback** [comment]
can you merge master and ping on green.

**Comment by cleconte987** [comment]
@jreback 
Merged

File: scripts/validate_rst_title_capitalization.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "Arrow",
     "Parquet",
     "MultiIndex",
+    "MultiIndexing",
```
Comment (WillAyd) [inline scripts/validate_rst_title_capitalization.py:45]: I think this should just be `multi-indexing` when referenced instead

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: scripts/validate_rst_title_capitalization.py (hunk 1 above)
Comment (cleconte987) [reply scripts/validate_rst_title_capitalization.py]: Ok I removed it, and changed occurrences to 'Multi-indexing'

**Comment by cleconte987** [comment]
Im not sure to understand why there are unsuccessful tests here

**Review by WillAyd — APPROVED** [review]
lgtm

File: doc/source/whatsnew/v0.16.0.rst [hunk 2]
```diff
@@ -218,7 +218,7 @@ Backwards incompatible API changes
 
 .. _whatsnew_0160.api_breaking.timedelta:
 
-Changes in Timedelta
+Changes in timedelta
```
Comment (jbrockmendel) [inline doc/source/whatsnew/v0.16.0.rst:221]: should this stay capitalized?

**Comment by jreback** [comment]
thanks!

**Pull request merged**